eng/InternalTools.props

```diff
@@ -3,7 +3,11 @@
 <Project>
   <Import Project="dependencies.props" />
 
-  <ItemGroup>
+  <PropertyGroup>
+    <UsingILCTools Condition="$(TargetGroup.EndsWith('aot'))">true</UsingILCTools>
```

Comment:
nit: you could have also checked here if UsingILCTools is '' as overwriting that property if passed in globally logs an information message that it can't be overwritten.

Reply:
Fixed :smile: